.github/labeler.yml

```
@@ -0,0 +1,35 @@
+
+# label rules used by .github/workflows/label.yml
+
+# label 'ci' all automation-related steps and files
+#   Since this repository is in itself an automation process to deploy a server instance,
+#   we refer here to CI as the 'meta' configuration files for managing the code and integrations with the repository,
+#   not configurations related to the deployment process itself.
+CI:
+  - .*  # all '.<something>' files
+  - .github/workflows/*
+  - docs/Makefile
+  - pylintrc
+  - tox.ini
+  - Makefile
+
+docs:
+  - "*.rst"
+  - "*.example"
+  - ".readthedocs.yml"
+  - docs/**/*
+  - AUTHORS.rst
+  - CONTRIBUTING.rst
+  - ISSUE_TEMPLATE/**/*
+  - ISSUE_TEMPLATE.md
+  - PULL_REQUEST_TEMPLATE.md
+  - README.rst
+
+indicators:
+  - xclim/indicators/**/*
+  - xclim/indicators/**/*
+  - xclim/data/**/*.json
+  - xclim/data/**/*.yml
```

```suggestion
indicators:
  - xclim/indicators/**/*
  - xclim/indicators/**/*
  - xclim/indices/**/_*.py
  - xclim/data/**/*.json
  - xclim/data/**/*.yml
```
Suggestion to add the compute functions python files.